platforms/documentation/docs/src/docs/userguide/jvm/toolchains.adoc

```diff
@@ -313,6 +313,13 @@ In order to disable auto-provisioning, you can use the `org.gradle.java.installa
 * Either start gradle using `-Porg.gradle.java.installations.auto-download=false`
 * Or put `org.gradle.java.installations.auto-download=false` into a `gradle.properties` file.
 
+[NOTE]
+====
+After disabling the auto provisioning, you need to ensure that the specified JRE/JDK version in the build.gradle file is already installed locally. Then, you need to stop the Gradle daemon so that it can be reinitialized for the next build. You can use the `./gradlew --stop` command to stop the daemon process.
```

Comment:
```suggestion
After disabling the auto provisioning, ensure that the specified JRE/JDK version in the build file is already installed locally. 
Then, stop the Gradle daemon so that it can be reinitialized for the next build. 
You can use the `./gradlew --stop` command to stop the daemon process.
```

Reply:
I have updated the PR, could you help me review it again? How should I contact the JVM team?

Reply:
I have updated the PR, could you help me review it again? How should I contact the JVM team? @lkasso